inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -0,0 +1,166 @@
+                           c="protein_solv.gro",
+                           p="topol.top",
+                           o="ions.tpr")
+        self.simulator.run(input_dir=self.working_dir,
+                           output_directory=self.working_dir,
+                           method_name="genion",
+                           s="ions.tpr",
+                           o="protein_solv_ions.gro",
+                           p="topol.top",
+                           pname="NA",
+                           nname="CL",
+                           neutral="yes")
+        self.simulator.run(input_dir=self.working_dir,
+                           output_directory=self.working_dir,
+                           method_name="grompp",
+                           f="energy_minimization.mdp",
+                           c="protein_solv_ions.gro",
+                           p="topol.top",
+                           o="em.tpr")
+        self.simulator.run(input_dir=self.working_dir,
+                           output_directory=self.working_dir,
+                           method_name="mdrun",
+                           deffnm="em",
+                           v="yes")
+        self.simulator.run(input_dir=self.working_dir,
+                           output_directory=self.working_dir,
+                           method_name="grompp",
+                           f="simulation.mdp",
+                           c="em.gro",
+                           r="em.gro",
+                           p="topol.top",
+                           o="solvated_protein.tpr")
+        self.simulator.run(input_dir=self.working_dir,
+                           output_directory=self.working_dir,
+                           method_name="mdrun",
+                           deffnm="solvated_protein",
+                           v="yes")
+
+    def setup_working_dir(self, working_dir, protein_pdb):
+        """Setup the working directory for the simulation. 
+        For this scenario, the input and output directories are the same."""
+
+        # Create working_dir if it does not exist
+        if working_dir is None:
+            scenario_name_splitted = split_camel_case(self.__class__.__name__)
+            working_dir_prefix = "-".join(scenario_name_splitted).lower()
+            working_dir = get_timestamped_path(f"{working_dir_prefix}-output")
+        working_dir = resolve_path(working_dir)
+        if not os.path.exists(working_dir):
+            os.makedirs(working_dir)
+
+        # Copy protein pdb to working_dir
+        shutil.copy(protein_pdb,
+                    os.path.join(working_dir, os.path.basename(protein_pdb)))
+
+        # Copy template files to working_dir
+        for file in os.listdir(self.template_dir):
+            shutil.copy(os.path.join(self.template_dir, file),
+                        os.path.join(working_dir, file))
+
+        # Remove all files that have .jinja in the working_dir
+        remove_files_with_tag(working_dir, ".jinja")
```

Comment:
Why is this necessary?

Reply:
This removes the .jinja files from the working dir. This files are necessary to create the mdp files filled out with the parameters defined by the user.